students/unsan-Jeong/users/views.py

```diff
@@ -0,0 +1,41 @@
+import json ,re
+
+from django.http     import JsonResponse, HttpResponse
+from django.views    import View
+
+#from django.core.exceptions import ValidationError
+
+from users.models import Users
+
+class SignUp(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        password_length = 8
+        try:
+            if len(data['password']) < password_length:  
+                return JsonResponse({'MESSAGE':'password of at least eight characters'}, status=404)
+            
+            if (re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', data['email']) == None):
```

Comment:
```suggestion
            if (re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', data['email']) is None):
```
None을 비교 할 때는 is를 더 많이 사용합니다.